code/modules/cargo/exports/weapons.dm

```diff
@@ -155,7 +155,7 @@
 	export_types = list(/obj/item/grenade/chem_grenade/large)
 
 /datum/export/weapon/gravworm
-	cost = 150
+	cost = 1000
```

Comment:
### _takes a big sip of ice-cold cherry coca cola_
### aaaaaahhh.
### what do we have here, hm?
[Did you...](https://youtu.be/u0jq42xgTzo)
[Did you think we wouldn't notice?](https://youtu.be/rJrxxnBWPdI)
[Did you think it'd get past us?](https://youtu.be/xEvJmI7uJ-g)

[You'll be needing one of these.](https://large-type.com/#%2F%2Fskyrat%20edit)